src/puppet-web/event.ts

```diff
@@ -244,6 +244,10 @@ async function onServerLogin(this: PuppetWeb, data, attempt = 0): Promise<void>
   // if `login` event fired before this.bridge inited, we delay the event for 1 second.
   if (!this.bridge) {
     log.verbose('PuppetWebEvent', 'onServerLogin() fired before bridge inited. delay for 1 second.')
+    if (!arguments) {
+      log.warn('PuppetWebEvent', 'onServerLogin() arguments is null')
+      throw new Error('onServerLogin() arguments is null')
```

Comment:
When I use the cookie file to run again, I do not understand why it is still running after it is thrown

Reply:
Please use `args` to replace the `arguments` in the code, then I believe we need not check `!arguments` again because we can get an empty array for `args`.